warp_drive/training/trainer.py

```diff
@@ -631,9 +605,16 @@ def _bookkeep_rewards_and_done_flags(self, batch_index):
             rewards = self.cuda_envs.cuda_data_manager.data_on_device_via_torch(
                 _REWARDS
             )
-            self.cuda_envs.cuda_data_manager.data_on_device_via_torch(
-                name=f"{_REWARDS}_batch"
-            )[batch_index] = rewards
+            for policy in self.policies:
+                if len(self.policies) > 1:
+                    agent_ids_for_policy = self.policy_tag_to_agent_id_map[policy]
+                    self.cuda_envs.cuda_data_manager.data_on_device_via_torch(
+                        name=f"{_REWARDS}_batch_{policy}"
```

Comment:
_REWARDS -> _REWARDS_NAME_STRING? 

Clarify that this constant is a string label?